recipe/meta.yaml

```diff
@@ -11,21 +11,19 @@ source:
   sha256: 931378d93d11b298cf511dd634cf5ea4cb249a28ef84160b3247ee9afb4e8ab0
 
 build:
+  skip: True  # [py<36]
```

Comment:
well, if it is noarch, then there is the noarch: attribute missing.  But in general we can stay in sync with CF here, so building it arch-sepcific is fine IMHO here